### ISSUE TITLE
inject gRPC into all missions

### ISSUE THREAD

**Issue by rkusa** [DCS-gRPC/rust-server#32]
Here is a hook that can be used to inject the gRPC server into all missions of the DCS server:

`Saved Games\DCS.openbeta\Scripts\Hooks\gRPC.lua`:

```lua
local function inject()
  log.write("[gRPC]", log.INFO, "Injecting gRPC into mission")

  local result = net.dostring_in("server", [[
    if GRPC == nil then
      package.cpath = package.cpath.."M:\\Development\\DCS-gRPC\\rust-server\\target\\debug\\?.dll;"
      GRPC = { basePath = "M:\\Development\\DCS-gRPC\\rust-server\\lua\\" }
      dofile(GRPC.basePath .. "grpc.lua")
    end
  ]])
end

local function init()
  local handler = {}

  function handler.onSimulationResume()
    local ok, err = pcall(inject)
    if not ok then
      log.write("[gRPC]", log.ERROR, "Failed to inject: "..tostring(err))
    end
  end

  DCS.setUserCallbacks(handler)
end

local ok, err = pcall(init)
if not ok then
  log.write("[gRPC]", log.ERROR, "Failed to init: "..tostring(err))
end
```

Adding it as an issue here to keep track of it. We can think of somehow adding it to the repo in the future.

**Comment by rurounijones** [comment]
How do you see this working with things like `enableEval()` or allowing the IP / port to be set? It should be done in the hook? A separate config file?

**Comment by rurounijones** [comment]
Implemented by https://github.com/DCS-gRPC/rust-server/pull/75

**Comment by rurounijones** [comment]
Released in 0.2.0
